pkg/proxy/iptables/proxier.go

```diff
@@ -601,14 +601,14 @@ func servicePortEndpointChainName(servicePortName string, protocol string, endpo
 // TODO: move it to util
 func (proxier *Proxier) deleteEndpointConnections(connectionMap []proxy.ServiceEndpoint) {
 	for _, epSvcPair := range connectionMap {
-		if svcInfo, ok := proxier.serviceMap[epSvcPair.ServicePortName]; ok && svcInfo.GetProtocol() == v1.ProtocolUDP {
+		if svcInfo, ok := proxier.serviceMap[epSvcPair.ServicePortName]; ok && (svcInfo.GetProtocol() == v1.ProtocolUDP || svcInfo.GetProtocol() == v1.ProtocolSCTP) {
```

Comment:
Might make sense to create a separate function like "needsClearConntrack(proto v1.Protocol) bool" for this and use it in both places here, *and* in proxier_test.go below.